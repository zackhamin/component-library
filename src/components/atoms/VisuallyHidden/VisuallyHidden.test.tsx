import { VisuallyHiddenSpan } from '.';
import { render, screen } from '@testing-library/react';

describe('VisuallyHiddenSpan renders correctly', () => {
	it('renders correctly', () => {
		const { container } = render(
			<VisuallyHiddenSpan>Hidden Span</VisuallyHiddenSpan>
		);
		expect(container).toMatchSnapshot();
	});
	it('Hidden span is not visible', () => {
		render(<VisuallyHiddenSpan>Hidden Span</VisuallyHiddenSpan>);
		const span = screen.queryByText('Hidden Span');
		expect(span).toBeVisible();
	});
});
