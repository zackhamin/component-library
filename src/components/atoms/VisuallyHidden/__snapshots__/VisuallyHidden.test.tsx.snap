// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisuallyHiddenSpan renders correctly renders correctly 1`] = `
.c0 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
  white-space: nowrap;
  word-wrap: normal;
}

<div>
  <span
    class="c0"
  >
    Hidden Span
  </span>
</div>
`;
